src/braket/simulator/braket_simulator.py

```diff
@@ -63,3 +63,33 @@ def run(
     @abstractmethod
     def properties(self) -> DeviceCapabilities:
         """DeviceCapabilities: Properties of the device."""
+
+
+class MultiSimulator(BraketSimulator):
```

Comment:
can we also export this `MultiSimulator` from `__init__.py` in this module? to allow
`from braket.simulator import MultiSimulator`
instead of having to do
`from braket.simulator.braket_simulator import MultiSimulator`